docs/contributing/gatsby-style-guide.md

```diff
@@ -147,7 +147,7 @@ If you want to abbreviate a term in your article, write it out fully first, then
 put the abbreviation in parentheses. After that, you may use the abbreviation going for the rest of the article. For example, "In computer science, an
 abstract syntax tree (AST) is ..."
 
-### Use SEO optimized titles
+### Use Searcg Engine Optimization (SEO) optimized titles
```

Comment:
```suggestion
### Use Search Engine Optimization (SEO) optimized titles
```

Had a different thought, as well. Could just say, "Optimize titles for search engines"? And then define SEO acronym later down if it's there.

Reply:
That's a good thought. I'll see how that could work.